### PR TITLE
Feature/cosmetics 31

### LINKED ISSUE #31: Cosmetics
- [x] Move help button in stepper (only show in current step)
- [x] Move Next/Prev buttons to the bottom
- [x] Remove share button (EST)
- [x] List per page = 20

### PR DESCRIPTION
**Pull request by matiasgarcia91**
### :pushpin: References

* **Issue:** Closes #31 

### :art: 
image
image

### CODE REVIEW

File: src/components/wizard/Wizard.jsx [hunk 1]
```diff
@@ -172,7 +177,11 @@ class Wizard extends React.Component {
 
         return (
             <div className={classes.root}>
-                <Stepper nonLinear={true} activeStep={currentStepIndex}>
+                <Stepper
+                    nonLinear={true}
+                    activeStep={currentStepIndex}
+                    style={{ marginRight: 10, marginLeft: 10 }}
```
Comment (tokland) [inline src/components/wizard/Wizard.jsx]: Same than the other PR, move to static styles.

File: src/components/campaign-wizard/FormHeading.jsx [hunk 2]
```diff
@@ -9,7 +9,12 @@ import Icon from "@material-ui/core/Icon";
 function FormHeading({ variant, title, onBackClick }) {
     return (
         <div>
-            <IconButton onClick={onBackClick} color="secondary" aria-label={i18n.t("Back")}>
+            <IconButton
+                onClick={onBackClick}
+                color="secondary"
+                aria-label={i18n.t("Back")}
+                style={{ marginBottom: 7 }}
```
Comment (adrianq) [inline src/components/campaign-wizard/FormHeading.jsx]: This style still being used inline. @matiasgarcia91 can you please extract it to a class?

Comment (matiasgarcia91) [reply src/components/campaign-wizard/FormHeading.jsx]: Is a constant style object enough? This is a functional component not wrapped with the material-ui `withStyles`

**Comment by adrianq** [comment]
@tokland @matiasgarcia91 I have created an [issue](https://github.com/EyeSeeTea/vaccination-app/issues/50) to decide what to do with the new confirmation dialog Matias just added

File: src/components/campaign-wizard/FormHeading.jsx (hunk 2 above)
Comment (tokland) [reply src/components/campaign-wizard/FormHeading.jsx]: Yes, we tend to use withStyles for new components, but using a styles object is also fine.

**Pull request merged**